infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration scan="true" scanPeriod="30 seconds">
 
+    <!-- https://logback.qos.ch/manual/configuration.html#stopContext -->
+    <shutdownHook/>
```

Comment:
Could you explain this line?

Reply:
"In standalone Java applications, adding a <shutdownHook/> directive to your configuration file is an easy way to ensure that any ongoing compression tasks are allowed to finish before JVM exit."